sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -41,132 +41,142 @@ public DocumentModelAdministrationLiveTests(bool isAsync, DocumentAnalysisClient
+            try
+            {
+                operation = await client.BuildDocumentModelAsync(WaitUntil.Started, trainingFilesUri, buildMode, modelId, prefix, options);
+
+                await (buildMode == DocumentBuildMode.Neural
+                    ? operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1))
+                    : operation.WaitForCompletionAsync());
+            }
+            finally
+            {
+                if (operation != null && operation.HasValue)
+                {
+                    await client.DeleteDocumentModelAsync(modelId);
+                }
+            }
 
             Assert.IsTrue(operation.HasValue);
 
             DocumentModelDetails model = operation.Value;
 
-            ValidateDocumentModelDetails(model);
-
-            Assert.AreEqual(1, model.DocumentTypes.Count);
-            Assert.IsTrue(model.DocumentTypes.ContainsKey(modelId));
-
-            DocumentTypeDetails documentType = model.DocumentTypes[modelId];
+            Assert.AreEqual(modelId, model.ModelId);
+            Assert.AreEqual(options.Description, model.Description);
+            Assert.AreEqual(ServiceVersionString, model.ApiVersion);
+            Assert.Greater(model.CreatedOn, startTime);
 
-            Assert.AreEqual(DocumentBuildMode.Neural, documentType.BuildMode);
-        }
+            if (_serviceVersion >= DocumentAnalysisClientOptions.ServiceVersion.V2023_02_28_Preview)
+            {
+                Assert.Greater(model.ExpiresOn, model.CreatedOn);
+            }
+            else
+            {
+                // We have changed the following validation because of a service bug. This needs to be updated once the bug is fixed.
+                // More information: https://github.com/Azure/azure-sdk-for-net/issues/35809
 
-        [RecordedTest]
-        public async Task BuildDocumentModelSucceedsWithValidPrefix()
-        {
-            var client = CreateDocumentModelAdministrationClient();
-            var trainingFilesUri = new Uri(TestEnvironment.BlobContainerSasUrl);
-            var modelId = Recording.GenerateId();
+                // The expected behavior. This must be added back once the service bug is fixed.
+                // Assert.IsNull(model.ExpiresOn);
 
-            BuildDocumentModelOperation operation = await client.BuildDocumentModelAsync(WaitUntil.Completed, trainingFilesUri, DocumentBuildMode.Template, modelId, "subfolder/");
+                // The current behavior. This assertion must be removed once the service bug is fixed.
+                Assert.Greater(model.ExpiresOn, model.CreatedOn);
+            }
 
-            Assert.IsTrue(operation.HasValue);
-            Assert.IsNotNull(operation.Value.ModelId);
-        }
+            CollectionAssert.AreEquivalent(options.Tags, model.Tags);
 
-        [RecordedTest]
-        [Ignore("https://github.com/azure/azure-sdk-for-net/issues/28272")]
-        public void BuildDocumentModelFailsWithInvalidPrefix()
```

Comment:
Test renamed to `BuildDocumentModelThrowsWhenTrainingFilesAreMissing` to make intent more clear.